python/ml4ir/features/feature_config.py

```diff
@@ -294,12 +317,41 @@ def generate_mask(self):
         return {
             "name": "mask",
             "trainable": False,
-            "dtype": "float",
+            "dtype": self.get_rank("dtype"),
             "feature_layer_info": {"type": FeatureTypeKey.NUMERIC, "shape": None},
             "serving_info": {"name": "mask", "required": False},
             "tfrecord_type": TFRecordTypeKey.SEQUENCE,
         }
 
+    def create_dummy_sequence_example(self, num_records=1, required_only=False):
```

Comment:
@jakemannix Added functionality to create a dummy sequence example with given number of records.

Reply:
Would be nice to allow passing in of a dict of feature values to put in the SequenceExample :)

Reply:
This story has already been so far away from 1SP, I don't want to scope creep it. I will add a new work item to do this if that's fine with you.

Reply:
Created new work item -> https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000007QMb2IAG/view

It's a simple change. But we might want to use this ability to add more unit tests and update the model training tests to check for weights and loss convergence.

Reply:
for future reference: links to internal GUS work items won't be helpful in the open source world ;P